tests/results/test_disability_observer.py

```diff
@@ -69,7 +69,7 @@ def test_disability_observer_setup(mocker):
         additional_stratifications=observer.config.include,
         excluded_stratifications=observer.config.exclude,
         when="time_step__prepare",
-        formatter=observer.formatter,
+        results_formatter=observer.formatter,
```

Comment:
Nit: you reordered them all except this one, though it doesn't actually matter since they're all keyword arguments.

Reply:
Yeah I just reordered b/c ocd :/